packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -233,11 +250,34 @@ export default {
                     callbacks: {
                         handleContentCards: this.metadataContentCards
                     },
-                    loggerCallbacks: {
-                        logger: this.handleLogging(this.$logger)
-                    }
+                    logger: this.$logger,
+                    tags: this.tags
                 });
+
+                this.$logger.logInfo(
+                    `Content Cards (setupMetadata) - BrazeAdapter successfully initialised. Key: (${this.loggingKey})`,
+                    this.$store,
```

Comment:
Not sure how much data is in `this.$store` and how/where it gets sent, just a note on being sensible.

Reply:
Also there seems to be some repetition in the log data, have you considered extracting that to reduce repetition?

Reply:
Thanks for the comments, on point 1 its a requirement of the logger plugin to pass the whole store object, so I have to do that. point 2 agree I will make the abstraction.